assets/js/modules/sign-in-with-google/components/common/AnyoneCanRegisterDisabledNotice.js

```diff
@@ -61,6 +57,12 @@ export default function AnyoneCanRegisterDisabledNotice( { className } ) {
 		select( CORE_SITE ).getAdminSettingsURL()
 	);
 
+	const { dismissItem } = useDispatch( CORE_USER );
+
+	const dismissNotice = useCallback( () => {
```

Comment:
If we are going to use `useCallback` here, we need to pass an empty array as the second argument.

Reply:
@eugene-manuilov  I removed it and called it directly in an arrow function against the `dismissButton` onClick as per your previous feedback.